### PR TITLE
EWK Gamma+jets LO cards for UL

### PR DESCRIPTION
**Pull request by alpakpinar**
Hello, I am working on producing LO EWK gamma+jets sample for UL. In the genproductions repo, I couldn't find the corresponding cards for the ReReco LO EWK gamma+jets sample in [1] (sorry if I missed it somehow). Based on the gridpack used in this sample, I prepared the Madgraph cards in this PR. Do you think these cards look good for LO EWK gamma+jets UL production? Thanks a lot for the help and comments!

[1]: https://cms-pdmv.cern.ch/mcm/requests?prepid=SMP-RunIIFall17wmLHEGS-00009&page=0&shown=127

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AJJ_EWK_SM_5f_LO/AJJ_EWK_SM_5f_LO_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+  .false.     = gridpack  !True = setting up the grid pack
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  250       = nevents ! Number of unweighted events requested
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AJJ_EWK_SM_5f_LO/AJJ_EWK_SM_5f_LO_run_card.dat:32]: increase this to 3000

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AJJ_EWK_SM_5f_LO/AJJ_EWK_SM_5f_LO_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+#*********************************************************************
+#
+#*********************************************************************
+# Store info for systematics studies                                 *
+# WARNING: If use_syst is T, matched Pythia output is                *
+#          meaningful ONLY if plotted taking matchscale              *
+#          reweighting into account!                                 *
+#*********************************************************************
+   T  = use_syst      ! Enable systematics studies
+   F  = mc_grouped_subproc
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AJJ_EWK_SM_5f_LO/AJJ_EWK_SM_5f_LO_run_card.dat:270]: Is this necessary?
I am not familiar with this option but it seems that it makes MadGraph run slower than usual according to Qiang [1]

[1] https://github.com/cms-sw/genproductions/issues/1220

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AJJ_EWK_SM_5f_LO/AJJ_EWK_SM_5f_LO_run_card.dat (hunk 1 above)
Comment (alpakpinar) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AJJ_EWK_SM_5f_LO/AJJ_EWK_SM_5f_LO_run_card.dat]: I increased this option to 3000 in the most recent commit

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AJJ_EWK_SM_5f_LO/AJJ_EWK_SM_5f_LO_run_card.dat (hunk 2 above)
Comment (alpakpinar) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AJJ_EWK_SM_5f_LO/AJJ_EWK_SM_5f_LO_run_card.dat]: I think it's not really necessary indeed, we earlier removed this option from the EWK W and Z cards as well. In the latest commit, I removed this option, thanks for looking into these cards!

**Comment by sihyunjeon** [comment]
@agrohsje Looks fine to me

**Comment by alpakpinar** [comment]
Thanks a lot for taking a look into the cards! Please let me know if further updates are needed and I can make the necessary modifications.

**Comment by alpakpinar** [comment]
Hello @sihyunjeon, @agrohsje, are there any news on these cards? Let me know if I can do something, thank you!

**Comment by sihyunjeon** [comment]
@agrohsje Waiting for Alexander's approval. I am fine with the cards

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AJJ_EWK_SM_5f_LO/AJJ_EWK_SM_5f_LO_run_card.dat [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+#*******************                                                 
+# Standard Cuts
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Minimum and maximum pt's (for max, -1 means no cut)                *
+#*********************************************************************
+ 25  = ptj       ! minimum pt for the jets 
+ 25  = ptb       ! minimum pt for the b 
+ 50  = pta       ! minimum pt for the photons
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AJJ_EWK_SM_5f_LO/AJJ_EWK_SM_5f_LO_run_card.dat:110]: Looks good to me. What is the analysis level cut for the photon. Is 50 well enough below this threshold?

**Pull request merged**